core/src/main/java/com/sequenceiq/cloudbreak/controller/validation/ParametersValidator.java

```diff
@@ -29,25 +29,31 @@
     @Inject
     private ErrorHandlerAwareReactorEventFactory eventFactory;
 
-    public void validate(String platform, CloudCredential cloudCredential, Map<String, String> parameters, String userId, Long workspaceId) {
+    public ParametersValidationRequest triggerValidate(String platform, CloudCredential cloudCredential, Map<String, String> parameters, String userId,
+            Long workspaceId) {
         if (parameters == null || parameters.isEmpty()) {
-            return;
+            return null;
```

Comment:
I don't see where it is used, but is it a good design to return null?

Reply:
The response of this method is used in this method (currently). Anyway pass a `null` for a non nullable argument is also not a nice patter (but used everywhere).